MinVer/build/MinVer.targets

```diff
@@ -46,6 +46,7 @@
       <FileVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).0</FileVersion>
       <PackageVersion>$(MinVerVersion)</PackageVersion>
       <Version>$(MinVerVersion)</Version>
+      <NoWarn>$(NoWarn);NU5015</NoWarn>

```

Comment:
This `PropertyGroup` is inside the `Target`, which isn't what we want. Instead, if you move it up to the one on lines 8-11, and that should be perfect.

Reply:
@bording I think I also prefer it in the 8-11 block but, out of interest, would it not work inside the target?

Reply:
I'd have to test it to be sure, but I suspect it wouldn't because the properties in the target would only be evaluated once the target was executed, and that could be too late to influence the warning settings. 

Then even if it did work when running a build from the command line, depending on what runs during a design-time build, it might not work from inside VS.